### PR TITLE
Doc: Update onboarding.mdx

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR has the following updates:

1. `getting-started-overview.mdx`
- Adds Headless CMS hyperlink.
- Adds a short description for each topic of the track.

2. `onboarding.mdx`
- Fixes image's path (@renatamottam, I found out how to use file manager to upload images 👀 )
- Adds missing Hyperlinks

[Visit Preview](https://faststore-site-git-fix-onboarding-faststore.vercel.app/)

### CODE REVIEW

File: apps/site/pages/docs/getting-started/getting-started-overview.mdx [hunk 1]
```diff
@@ -17,18 +17,25 @@ This documentation is currently under development.
 
 In this Getting Started track, you'll build and deploy a FastStore Storefront at the domain `https://<account-name>.vtex.app/`.
 
-By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS](/tbd), [Catalog](/tbd), and [Checkout](/tbd).
+By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS]([/tbd](https://www.faststore.dev/tutorials/cms/0)).
```
Comment (gvc) [inline apps/site/pages/docs/getting-started/getting-started-overview.mdx:20]: Is this `[/tbd]` intentional?

File: apps/site/pages/docs/getting-started/onboarding.mdx [hunk 2]
```diff
@@ -51,15 +51,15 @@ Now that you have an overview of the app, let's start the project.
 2. In the **Setup & Create Project** page, ensure to define the basic configuration for your store:
 
 #### CMS Integration
-The option available to use a CMS in a FastStore project is [VTEX Headless CMS](/tbd), a VTEX'S Content Management solution.
+The option available to use a CMS in a FastStore project is [VTEX Headless CMS](https://www.faststore.dev/tutorials/cms/0), a VTEX'S Content Management solution.
```
Comment (gvc) [inline apps/site/pages/docs/getting-started/onboarding.mdx:54]: I think that there's an extra `a` and the s after the apostrophe should be lower case

```suggestion
The option available to use a CMS in a FastStore project is [VTEX Headless CMS](https://www.faststore.dev/tutorials/cms/0), VTEX's Content Management solution.
```

File: apps/site/pages/docs/getting-started/onboarding.mdx [hunk 3]
```diff
@@ -51,15 +51,15 @@ Now that you have an overview of the app, let's start the project.
...
 
 | Field | Description |
 | ------ | ----------- |
 | Title Tag | Type a keyword-optimized title for your store. For example: `Buy Online with the Best Game Store | GameStore.`. In this example, we used the length limit of around 50-60 characters to ensure it is fully displayed in search results without being truncated. It is also written in a way that accurately describes the webpage's content. |
 | Description Tag | Type a short description about your store, for example `Shop online at GameStore, the best game store for PC, Xbox, PS4 and Headphones. Explore our wide selection and buy your favorite games today!` |
-| Title Template | [TBD] For example `Headphones | GameStore`.|
+| Title Template | For example `Headphones | GameStore`.|
 | Site Author| type the name of the agency or person responsible for the storefront. |
 
 #### Github Acesss
```
Comment (gvc) [inline apps/site/pages/docs/getting-started/onboarding.mdx:65]: After the `Catalog` session, there's a section starting with `3.` that seems to be out of place. Should we correct it in this PR?

File: apps/site/pages/docs/getting-started/getting-started-overview.mdx [hunk 4]
```diff
@@ -17,18 +17,25 @@ This documentation is currently under development.
...
-**1. [FastStore Onboarding](/tbd)** 
+  
+## 1. [FastStore Onboarding](/docs/getting-started/onboarding)
 
 Start your project with the FastStore Onboarding app. It streamlines the process by setting up the essential store configurations, creating a Github repository for your store, and providing a live demo preview of your store.
 
-**2. [Setting Up the Project](/tbd)**
+## 2. [Setting Up the Project](/docs/getting-started/settingup-project)
+
+Discover the core technologies we will use in this tutorial and set up all the necessary tools to implement your very first FastStore storefront and create your first storefront customization.
```
Comment (gvc) [inline apps/site/pages/docs/getting-started/getting-started-overview.mdx:29]:
```suggestion
Discover the core technologies that we will use in this tutorial and set up all the necessary tools to implement your very first FastStore storefront and create your first storefront customization.
```

**Review by gvc — CHANGES_REQUESTED** [review]
Small changes 🙏 So great to see our documentation coming to life

File: apps/site/pages/docs/getting-started/getting-started-overview.mdx (hunk 1 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/getting-started/getting-started-overview.mdx:20]: No, thanks, Guila!🙏 I'm going to commit the fix for it

File: apps/site/pages/docs/getting-started/onboarding.mdx (hunk 3 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/getting-started/onboarding.mdx:65]: Good! I removed the `3` and revised the sentence for conciseness.

File: apps/site/pages/docs/getting-started/getting-started-overview.mdx (hunk 4 above)
Comment (Mariana-Caetano) [reply apps/site/pages/docs/getting-started/getting-started-overview.mdx]: Thanks for the suggestion, Guila. I made a slight change to the sentence to have a clear message.

File: apps/site/pages/docs/getting-started/onboarding.mdx [hunk 5]
```diff
@@ -97,7 +95,7 @@ In this scenario, both the repository and the live preview of the store were suc
 | Customization| Access the documentation on how to customize your store. |
 | View Dashboard | Coming soon! |
 
-Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](/tbd).
+Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](https://evergreen.faststore.dev/docs/getting-started/settingup-project).
```
Comment (renatamottam) [inline apps/site/pages/docs/getting-started/onboarding.mdx:98]:
```suggestion
Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](/docs/getting-started/settingup-project).
```

File: apps/site/pages/docs/getting-started/onboarding.mdx [hunk 6]
```diff
@@ -112,4 +110,4 @@ Once the issue is resolved, you will see the [Repository and Live Store created]
 
 ## Next Step
 
-Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](/tbd).
\ No newline at end of file
+Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](https://evergreen.faststore.dev/docs/getting-started/settingup-project).
```
Comment (renatamottam) [inline apps/site/pages/docs/getting-started/onboarding.mdx:113]:
```suggestion
Now that your store is up and running, let's set up the development environment by following the step-by-step instructions in [Setting up the Project](/docs/getting-started/settingup-project).
```

**Review by renatamottam — APPROVED** [review]
Nice improvements!

**Pull request merged**